source/includes/stackpath/_workloads.md

```diff
@@ -398,25 +493,28 @@ Required | &nbsp;
 `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based. Can be either `VM` or `CONTAINER`.
 `image`<br/>*string* | The location of a Docker image to run as a container. Only editable when `type` is equal to `CONTAINER`.
 `specs`<br/>*string* | Specification type for resources which are allocated to each instance in a workload.
-`deploymentName`<br/>*string* | The name of the deployment. The deployment name must not exceed 18 characters.
-`deploymentPops`<br/>*Array[string]* | The points of presence of a deployment. In the regex format `[A-Z][A-Z][A-Z]`.
-`enableAutoScaling` <br/>*boolean* | Specify if you would like to enable autoscaling. If enabled, then `cpuUtilization` , `minInstancesPerPop` and `maxInstancesPerPop` are required else `deploymentInstancePerPops` is required.
+`deployments`<br/>*Array[Object]* | The list of deployment targets.
+`deployments.name`<br/>*string* | The name of the deployment.
+`deployments.pops`<br/>*Array[string]* | The points of presence of a deployment. In the regex format `[A-Z][A-Z][A-Z]`.
+`deployments.enableAutoScaling` <br/>*boolean* | Specifies if autoscaling is enabled. If enabled, then `cpuUtilization` , `minInstancesPerPop` and `maxInstancesPerPop` are required.
 
 Optional | &nbsp;
 ------- | -----------
-`environmentVariableKey`<br/>*string* | The key of the environmental variable you would like to edit. Only available when `type` is equal to `CONTAINER`.
-`environmentVariableValue`<br/>*string* | The value of the environmental variable you would like to edit. Only available when `type` is equal to `CONTAINER`.
-`secretEnvironmentVariableKey`<br/>*string* | The key of the secret environmental variable you would like to edit. Only available when `type` is equal to `CONTAINER`.
-`secretEnvironmentVariableValue`<br/>*string* | The value of the secret environmental variable you would like to edit. Only available when `type` is equal to `CONTAINER`.
+`environmentVariables` <br/>*Array[Object]* | A list of environment variables. Only applicable to workloads of `type` 'CONTAINER'.
+`environmentVariables.key` <br/>*string* | The location to obtain a value for an environment variable.
+`environmentVariables.value` <br/>*string* | An environment variable's value.
+`secretEnvironmentVariables` <br/>*Array[Object]* | A list of sensitive environment variables. Only applicable to workloads of `type` 'CONTAINER'.
+`secretEnvironmentVariables.key` <br/>*string* | The location to obtain a value for a secret environment variable.
+`secretEnvironmentVariables.value` <br/>*string* | A secret environment variable's value.l to `CONTAINER`.
```

Comment:
Fix this typo

Reply:
Fixed [here](https://github.com/cloudops/cloudmc-api-docs/pull/307/commits/dfd9768a21c2ec3faab74f6d17ec5881e7143f5f#diff-009b71b1c0eb5e3fe379e21d571a83b999e8661251e350ecb0efb9319d7a992eR520)